<launch>
    <node name="object_classifier" pkg="pamaral_object_grasping_pattern_recognition" type="object_classifier.py" output="screen">
        <param name="cnn_model_path" value="$(find pamaral_object_grasping_pattern_recognition)/models/cnn_model.h5"/>
        <param name="transformer_model_path" value="$(find pamaral_object_grasping_pattern_recognition)/models/transformer_model.h5"/>
    </node>
</launch>